interactive_engine/executor/ir/graph_proxy/src/utils/expr/eval_pred.rs

```diff
@@ -880,4 +1050,37 @@ mod tests {
             .eval_bool::<_, Vertices>(Some(&context))
             .unwrap());
     }
+
+    #[test]
+    fn test_predicates_to_condition() {
```

Comment:
Could you please add some `Not`, `And`, `Or` cases?

Reply:
added